### DISCUSSION TITLE
Getting error to start application

### DISCUSSION THREAD

**Discussion by aimanskie** [exoplatform/exo-community#73] [Q&A]
Hi Community,

Need help regarding this error when i tried to remove addons. Initially i removed exo-onlyoffice only and that was fine, exo was running. However, when i added more, from the logs uninstalling the addons was successful but i am getting this error like so when i added more in the list.
```
| ERROR | Called stop() on failed component [StandardEngine[Catalina].StandardHost[localhost]] to trigger clean-up but it failed too [org.apache.catalina.util.LifecycleBase<main>]
```
<img width="1162" alt="Screenshot at Mar 27 10-43-47" src="https://github.com/exoplatform/exo-community/assets/86446622/8ba15380-1042-4472-a917-d3ad1efc62f8">

<img width="913" alt="image" src="https://github.com/exoplatform/exo-community/assets/86446622/46dc3a35-a9e1-4fe2-85ae-771cc4c55589">

**Comment by ahamdi** [comment]
Hi @aimanskie 
FYI , some addons are related and one depends from the other.
seeing a part of the exception in the attached screenshot, I think the problem is related to Jitsi addon, which depends on exo-webconferencing addon .
You need to remove it too if you do not intend to have video calls in your platform, otherwise, you will have to reinstall exo-webconferencing.

**Reply by aimanskie** [reply]
Thanks! I think that did resolve that error. However, there's another error even when i removed the exo-web-conferencing  addon all together.

```
exo_home-exo-1    | 2024-03-27 18:04:33,194 | ERROR | Servlet.service() for servlet [portal] in context with path [/portal] threw exception [o.a.c.c.C.[.[.[/portal].[portal]<http-nio-0.0.0.0-8080-exec-2>]
exo_home-exo-1    | java.lang.NullPointerException: Cannot invoke "org.exoplatform.services.jcr.ext.app.SessionProviderService.getSessionProvider(Object)" because "providerService" is null
exo_home-exo-1    |     at org.exoplatform.frameworks.jcr.web.ThreadLocalSessionProviderInitializedFilter.doFilter(ThreadLocalSessionProviderInitializedFilter.java:138)
```

<img width="1154" alt="image" src="https://github.com/exoplatform/exo-community/assets/86446622/41e5c82e-2cbf-49f8-8268-7924f651ba2b">

**Reply by murthdaddy** [reply]
I'm also receiving similar NullPointerException error message on a new install...any help greatly appreciated!

<img width="1376" alt="image" src="https://github.com/exoplatform/exo-community/assets/168120613/80a56de7-cc68-493d-92b2-b999b880ff02">

**Reply by Avinash0729** [reply]
@murthdaddy - did u resolved this , iam facing same issue

**Reply by murthdaddy** [reply]
I @Avinash0729 I did not, so I just did a reinstall on fresh Ubuntu instance and now all working.  I'm now trying to get email integration working with gmail as per the docs, but struggling with that now :(

**Reply by Avinash0729** [reply]
@murthdaddy -which java version ur using and how u configured java_home ?

**Reply by murthdaddy** [reply]
<img width="913" alt="image" src="https://github.com/exoplatform/exo-community/assets/168120613/3e31504b-968b-49d3-9d4d-ecc2d0b72eb7">

**Comment by Avinash0729** [comment]
![Uploading image.png…]()

**Comment by Mohands84** [comment]
Hello All , 
Try to install exo-community:7.0.0 local . getting this error
exo-1       | 2025-05-06 11:28:04,383 | ERROR | Error while unlocking the locked nodes [o.e.s.cms.lock.impl.LockServiceImpl<main>]
exo-1       | javax.jcr.RepositoryException: Repository 'repository' not found.
exo-1       |   at org.exoplatform.services.jcr.impl.RepositoryServiceImpl.getRepository(RepositoryServiceImpl.java:208)
exo-1       |   at org.exoplatform.services.jcr.impl.RepositoryServiceImpl.getDefaultRepository(RepositoryServiceImpl.java:197)
exo-1       |   at org.exoplatform.services.jcr.impl.RepositoryServiceImpl.getCurrentRepository(RepositoryServiceImpl.java:190)
exo-1       |   at org.exoplatform.services.cms.lock.impl.LockServiceImpl.removeLocks(LockServiceImpl.java:241)
exo-1       |   at org.exoplatform.services.cms.lock.impl.LockServiceImpl.start(LockServiceImpl.java:132)
exo-1       |   at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)

**Reply by rdenarie** [reply]
Hi @Mohands84 
Can you create an account on community.exoplatform.org, and create a discussion here ? 
You will have more visibility for eXoers who can answers you.

Don't hesitate to post full startup logs, and your context.

Thanks
